### PR TITLE
feat: Move chunking to core

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**

### CODE REVIEW

File: Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+ */
+
+@testable import InfomaniakCore
+import XCTest
+
+/// Integration Tests of the RangeProviderGuts
+final class ITRangeProviderGuts: XCTestCase {
+    // MARK: - readFileByteSize
+
+    let file = "Matterhorn_as_seen_from_Zermatt,_Wallis,_Switzerland,_2012_August,Wikimedia_Commons"
```
Comment (adrien-coye) [inline Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift:26]: It looks like the file "Matterhorn_as_seen_from_Zermatt…" was not copied with the integration tests. They are probably failiing

**Review by adrien-coye — CHANGES_REQUESTED** [review]
I think you missed a sample file.

File: Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift (hunk 1 above)
Comment (PhilippeWeidmann) [reply Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift:26]: We already had in under /Resources for other tests

Comment (adrien-coye) [reply Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift:26]: Perfect 🤝

**Review by adrien-coye — APPROVED** [review]
All good. Thanks.

**Pull request merged**